assets/src/blocks/Happypoint/HappypointFrontend.js

```diff
@@ -64,8 +64,6 @@ export const HappypointFrontend = ({
                 <iframe
                   src={url}
                   cellSpacing={0}
-                  frameBorder={0}
```

Comment:
This two props are marked as `deprectated`.
![Uploading Screenshot 2024-01-23 at 10.29.54.png…]()